Library/Formula/tesseract.rb

```diff
@@ -24,10 +25,21 @@ class Tesseract < Formula
   end
 
   option "all-languages", "Install recognition data for all languages"
+  option "with-training-tools", "Install OCR training tools"
+  option "with-opencl", "Enable OpenCL support"
 
   depends_on "libtiff" => :recommended
   depends_on "leptonica"
 
+  if build.with? "training-tools"
```

Comment:
I see. I'm not super-opposed to the option, I guess, it's just the rather heavy dependencies involved.

Sadly, the way `brew` handles `if build.with?` dependencies is less than ideal and consequently we've been trying to get rid of `if build.with?` outside of `def install`. Essentially the option looks like it doesn't require anything extra under `brew info tesseract`:

```
==> Dependencies
Required: leptonica ✘
Recommended: libtiff ✔
==> Options
--all-languages
	Install recognition data for all languages
--with-opencl
	Enable OpenCL support
--with-training-tools
	Install OCR training tools
```

But obviously, there's actually a bunch of "silent" dependencies there when the option is invoked. I'll let the other maintainers weigh in on this particular use case and see what the consensus is.